test/test_structured_2d.jl

```diff
@@ -220,6 +220,30 @@ isdir(outdir) && rm(outdir, recursive=true)
       tspan = (0.0, 0.3))
   end
 
+  @trixi_testset "elixir_eulerpolytropic_convergence.jl" begin
```

Comment:
Did you check that this test and the other three each does not take more than 10s on your laptop or Roci to run (see also https://trixi-framework.github.io/Trixi.jl/stable/testing/#Adding-new-tests)?

Reply:
Executing the elixir after compilation takes 0.045603 seconds on my laptop. Does it take longer on other systems?

Reply:
No, this was just a general question :-) Sounds good!